### DISCUSSION TITLE
How do I free disk space occupied by vcpkg?

### DISCUSSION THREAD

**Discussion by proTerm** [microsoft/vcpkg#22034] [Q&A]
How do I free disk space occupied by `vcpkg`, without breaking the current builds ? I noticed it occupies a lot of disk space, is all this needed? What could I remove occasionally ?

**Comment by Osyotr** [comment]
`buildtrees` folder can safely be removed.
`downloads` may be cleaned (or moved out with asset cache)
Don't bother with `packages`, it will be repopulated every time you do `vcpkg install`
